src/components/reorderer/js/Reorderer.js

```diff
@@ -190,8 +190,7 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
             movables:    ".flc-reorderer-movable",
             selectables: ".flc-reorderer-movable",
             dropTargets: ".flc-reorderer-movable",
-            grabHandle: "",
```

Comment:
grabHandle shouldn't be removed.

Reply:
Actually i have just removed the comma from the grabHandle line and not the actual selector itself.

Reply:
sorry, my mistake.